src/sentry/static/sentry/app/views/organizationDiscover/utils.jsx

```diff
@@ -59,6 +59,11 @@ export function getOrderByOptions(queryBuilder) {
       }
     }
 
+    // Never allow ordering by project_name since this can't be done in Snuba
+    if (name === 'project_name') {
```

Comment:
This is only one item right now, but should we make it a list (and move it into `data`)?

Reply:
Same as above